### DISCUSSION TITLE
'BotFactory' is only available since Kotlin 1.4 and cannot be used in Kotlin 1.4

### DISCUSSION THREAD

**Discussion by LexieAlreadyTaken** [mamoe/mirai#1113] [Q&A]
在intellij idea里一个纯kotlin的项目中，想要运行的时候出现了这个问题。intellij idea只支持到Kotlin 1.4，不知道该怎么办。

**Comment by Him188** [comment]
更新 Kotlin 插件到最新版本
IntelliJ 使用 2020.1 及以上
编译使用 Kotlin 1.4.30

**Reply by LexieAlreadyTaken** [reply]
已经是这些配置了，还是不行，Kotlin插件最新是1.5，但Intellij IDEA好像不支持，只支持到1.4

**Reply by Karlatemp** [reply]
是 gradle/maven 吗？请检查 build.gradle(.kts) / pom.xml 里的 kotlin 插件版本
